airflow/cli/simple_table.py

```diff
@@ -62,6 +63,14 @@ def print_as_table(self, data: List[Dict]):
             table.add_row(*[str(d) for d in row.values()])
         self.print(table)
 
+    def print_as_plain_table(self, data: List[Dict]):
+        """Renders list of dictionaries as a simple table than can be easily piped"""
+        if not data:
+            self.print("No data found")
+            return
+        output = tabulate(data, tablefmt="plain")
```

Comment:
```suggestion
        output = tabulate(data, tablefmt="plain", headers='keys')
```
This change should do what I mentioned above. Referenced from the original implementation.